<!DOCTYPE html>
<!--[if lt IE 7 ]><html class="ie ie6" lang="en"> <![endif]-->
<!--[if IE 7 ]><html class="ie ie7" lang="en"> <![endif]-->
<!--[if IE 8 ]><html class="ie ie8" lang="en"> <![endif]-->
<!--[if (gte IE 9)|!(IE)]><!--><html lang="en"> <!--<![endif]-->
<head>

	<!-- Basic Page Needs
  ================================================== -->
	<meta charset="utf-8">
	<title>James C. Liao, Metabolic Engineering and Synthetic Biology Laboratory</title>
	<meta name="description" content="">
	<meta name="author" content="">

	<!-- Mobile Specific Metas
  ================================================== -->
	<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1">

	<!-- CSS & LINKS
  ================================================== -->
	<link rel="stylesheet" href="style.css">
	<link href='https://fonts.googleapis.com/css?family=Lato:400,700,700italic,400italic' rel='stylesheet' type='text/css'>
	<script type="text/javascript" src="jquery-1.3.2.min.js"></script>
	<link rel="shortcut icon" href="favicon.ico">


</head>


<body>
	<div id="top"></div>



	<div class="logo">

		<a href="index.html">
		<img src="images/logo.png" id="text"></a>


</div>

	<!-- Primary Page Layout
	================================================== -->

	<!-- Delete everything in this .container and get started on your own site! -->

		<div id="nav">
					<ul>
					<li><a href="index.html"><h1>News</h1></a></li>
					<li><a href="team.html"><h1>Team</h1></a></li>
					<li><a href="research.html"><h1>Research</h1></a></li>
					<li><a href="publications.html"><h1>Publications</h1></a></li>
					<li><a href="downloads.html"><h1>Downloads</h1></a></li>
					<li><a href="contact.html"><h1>Contact</h1></a></li>
					</ul>

		</div>

		<div id="sidebar">
			<a href="next-generation-biofuels.html"><h2>Next-generation Biofuels</h2></a><hr>
			<a href="obesity-and-fatty-acid-metabolism.html"><h2>Obesity & Fatty <br>Acid Metabolism</h2></a><hr>
			<a href="computational-analysis.html"><h2>Computational Analysis</h2></a><hr>
			<a href="gene-metabolic-circuits.html"><h2>Gene-metabolic Circuits</h2></a><hr>
			<a href="nitric-oxide-metabolism.html"><h2>Nitric Oxide Metabolism</h2></a><hr>
			</div>

		<div class="columnfull">
			

			<img src="images/titlelarge.png" style="margin:0px 0 40px 0;"></a><br><h4>Network Component Analysis</h4><br><br><p><i>Liao, J.C.; Boscolo, R; Yang, Y.L.; Tran,L.M.; Sabatti, C.; and Roychowdhury, V. (2003) <a href="http://www.seas.ucla.edu/~liaoj/Liao%20NCA.pdf">Network component analysis: Reconstruction of regulatory signals in biological systems</a> Proc. Natl. Acad. Sci. USA,  100: 15522–15527.<br> <a href="http://www.seas.ucla.edu/~liaoj/NCA_appendix/">Download Appendices here.</a></i><br><br>

			<p>  High-dimensional data sets generated by high-throughput technologies, such as DNA microarray, are often the outputs of complex networked systems driven by hidden regulatory signals. Traditional statistical methods for computing low-dimensional or hidden representations of these data sets, such as principal component analysis and independent component analysis, ignore the underlying network structures and provide decompositions based purely on a priori statistical constraints on the computed component signals. The resulting decomposition thus provides a phenomenological model for the observed data and does not necessarily contain physically or biologically meaningful signals. Here we develop a method, called network component analysis, for uncovering hidden regulatory signals from outputs of networked systems, when only a partial knowledge of the underlying network topology is available. The a priori network structure information is first tested for compliance with a set of identifiability criteria. For networks that satisfy the criteria, the signals from the regulatory nodes and their strengths of influence on each output node can be faithfully reconstructed. This method is first validated experimentally using the absorbance spectra of a network of various hemoglobin species. The method is then applied to microarray data generated from yeast Saccharamyces cerevisiae and the activities of various transcription factors during cell cycle are reconstructed by using recently discovered connectivity information for the underlying transcriptional regulatory networks.<br><br>

			 <i>Kao, K.C.; Yang, Y.; Boscolo, R.; Sabatti, C.;Roychowdhury, V. and Liao, J.C. (2004) <a href="http://www.seas.ucla.edu/~liaoj/Kao%20NCA%20PNAS.pdf">Transcriptome-based determination of multiple transcription regulator activities in Escherichia coli using network component analysis,</a> Proc. Natl. Acad. Sci. USA, 101:641-646.</i><br><br>
			 
			<p>Cells adjust gene expression profiles in response to environmental and physiological changes through a series of signal transduction pathways. Upon activation or deactivation, the terminal regulators bind to or dissociate from DNA, respectively, and modulate transcriptional activities on particular promoters. Traditionally, individual reporter genes have been used to detect the activity of the transcription factors.  This approach works well for simple, non-overlapping transcription pathways. For complex transcriptional networks, more sophisticated tools are required to deconvolute the contribution of each regulator. Here we demonstrate the utility of Network Component Analysis in determining multiple transcription factor activities based on transcriptome profiles and available connectivity information regarding network connectivity. We used Escherichia coli carbon source transition from glucose to acetate as a model system. Key results from this analysis were either consistent with physiology or verified using independent measurements.<br><br>

			<b>Generalized Network Component Analysis</b><br>
			
			<i>Tran, L.M.; M.P. Brynildsen; K. C. Kao; J. K. Suen and J. C. Liao (2005) gNCA: A framework for determining transcription factor activity based on transcriptome: Identifiability and numerical implementation, <a href="http://www.seas.ucla.edu/~liaoj/gNCA.pdf">Metabolic Engineering. </a>7 (2005) 128–141</i><br><br>
			 
			<p>Network Component Analysis (NCA) is a network structure-driven framework for deducing regulatory signal dynamics. In contrast to classical approaches such as principal component analysis or independent component analysis, NCA makes use of the connectivity structure from transcriptional regulatory networks to restrict the decomposition to a unique solution. However, the existing version of NCA cannot incorporate information beyond the network topology such as information obtained from regulatory gene knock-outs that constrain the dynamics of regulatory signals. The ability of incorporating such information enables a more accurate and self-consistent analysis over different experiments and extends NCA to systems that may not satisfy the identifiability criteria of NCA. In this paper, we derive a generalized form of NCA, gNCA, which significantly expands the capability of transcription network analysis by incorporating regulatory signal constraints arising from genetic knockouts. The theoretical bases including criteria for uniqueness of solution and distinguishability between networks are derived.  In addition, numerical techniques for robust decomposition are discussed. gNCA is then demonstrated using an Escherichia coli wild-type strain and an isogenic arcA deletion mutant during a carbon source transition.<br><br>

		
		

			<div style="display:block; height:100px;"></div>

		</div> <!-- END COLUMN -->


		<link href='https://fonts.googleapis.com/css?family=Lato:400,700,700italic,400italic' rel='stylesheet' type='text/css'>
<div class="footer">Site by <a href="http://www.studio-point.net">Studio Point</a></div>
	
<!--JS
================================================= -->


<!-- End Document
================================================== -->

</body>
</html>